src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="images/logo.png"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+            >
```

Comment:
It's good practice to provide the 'href' attribute for anchor tags, even if they are placeholders. You can use '#' as a temporary value.